kie-server-parent/kie-server-services/kie-server-services-common/src/main/java/org/kie/server/services/impl/ContainerManager.java

```diff
@@ -30,6 +30,10 @@
     private static final Logger logger = LoggerFactory.getLogger(ContainerManager.class);
 
     public void installContainers(KieServerImpl kieServer, Set<KieContainerResource> containers, KieServerState currentState, KieServerSetup kieServerSetup) {
+        installContainersSync(kieServer, containers, currentState, kieServerSetup);
+    }
+
+    public void installContainersSync(KieServerImpl kieServer, Set<KieContainerResource> containers, KieServerState currentState, KieServerSetup kieServerSetup) {
```

Comment:
the difference is here https://github.com/droolsjbpm/droolsjbpm-integration/blob/master/kie-server-parent/kie-server-remote/kie-server-jms/src/main/java/org/kie/server/jms/ContainerManagerEJB.java#L35 which is used on app servers do it can be done in async way but still valid for JEE environments